lib/compat/wordpress-6.5/class-wp-navigation-block-renderer.php

```diff
@@ -15,7 +15,7 @@
  */
 class WP_Navigation_Block_Renderer {
 
-	private static $list_item_nav_blocks = array(
+	private static $nav_blocks_wrapped_in_list_item = array(
 		'core/navigation-link',
 		'core/home-link',
 		'core/site-title',
```

Comment:
Why is this in _both_ of the lists? How can it both be wrapped and need a wrapper?

Reply:
It needs a wrapper, so once it has been wrapped, it is wrapped! 🤣